Source/swiftlint/Helpers/LintOrAnalyzeArguments.swift

```diff
@@ -44,6 +44,8 @@ struct LintOrAnalyzeArguments: ParsableArguments {
     var output: String?
     @Flag(help: "Show a live-updating progress bar instead of each file being processed.")
     var progress = false
+    @Flag(help: "Do not print deprecation warnings.")
+    var quietDeprecationWarnings = false
```

Comment:
Should this be a flag on the `SwiftLint.Lint` command since it's not referenced from the `SwiftLint.Analyze` command?

Also I'd prefer the name `silenceDeprecationWarnings` since they're completely silenced, not just "quieter". I assume you wanted to keep consistent naming with the `--quiet` flag, but that doesn't completely silence all logs, just logs less, so it's a bit different.

Reply:
I take both! 👍